src/utilities/filetypes/EpwFile.cpp

```diff
@@ -567,79 +568,271 @@ boost::optional<EpwDesignCondition> EpwDesignCondition::fromDesignConditionsStri
+  setCoolingEvaporationWetBulb0pt4(stringValues[index(EpwDesignField::CoolingEvaporationWetBulb0pt4)]);
+  setCoolingEvaporationMeanCoincidentDryBulb0pt4(stringValues[index(EpwDesignField::CoolingEvaporationMeanCoincidentDryBulb0pt4)]);
+  setCoolingEvaporationWetBulb1(stringValues[index(EpwDesignField::CoolingEvaporationWetBulb1)]);
+  setCoolingEvaporationMeanCoincidentDryBulb1(stringValues[index(EpwDesignField::CoolingEvaporationMeanCoincidentDryBulb1)]);
+  setCoolingEvaporationWetBulb2(stringValues[index(EpwDesignField::CoolingEvaporationWetBulb2)]);
+  setCoolingEvaporationMeanCoincidentDryBulb2(stringValues[index(EpwDesignField::CoolingEvaporationMeanCoincidentDryBulb2)]);
+  setCoolingMeanCoincidentWindSpeed0pt4(stringValues[index(EpwDesignField::CoolingMeanCoincidentWindSpeed0pt4)]);
+  setCoolingPrevailingCoincidentWindDirection0pt4(stringValues[index(EpwDesignField::CoolingPrevailingCoincidentWindDirection0pt4)]);
+  setCoolingDehumidificationDewPoint0pt4(stringValues[index(EpwDesignField::CoolingDehumidificationDewPoint0pt4)]);
+  setCoolingDehumidificationHumidityRatio0pt4(stringValues[index(EpwDesignField::CoolingDehumidificationHumidityRatio0pt4)]);
+  setCoolingDehumidificationMeanCoincidentDryBulb0pt4(stringValues[index(EpwDesignField::CoolingDehumidificationMeanCoincidentDryBulb0pt4)]);
+  setCoolingDehumidificationDewPoint1(stringValues[index(EpwDesignField::CoolingDehumidificationDewPoint1)]);
+  setCoolingDehumidificationHumidityRatio1(stringValues[index(EpwDesignField::CoolingDehumidificationHumidityRatio1)]);
+  setCoolingDehumidificationMeanCoincidentDryBulb1(stringValues[index(EpwDesignField::CoolingDehumidificationMeanCoincidentDryBulb1)]);
+  setCoolingDehumidificationDewPoint2(stringValues[index(EpwDesignField::CoolingDehumidificationDewPoint2)]);
+  setCoolingDehumidificationHumidityRatio2(stringValues[index(EpwDesignField::CoolingDehumidificationHumidityRatio2)]);
+  setCoolingDehumidificationMeanCoincidentDryBulb2(stringValues[index(EpwDesignField::CoolingDehumidificationMeanCoincidentDryBulb2)]);
+  setCoolingEnthalpy0pt4(stringValues[index(EpwDesignField::CoolingEnthalpy0pt4)]);
+  setCoolingEnthalpyMeanCoincidentDryBulb0pt4(stringValues[index(EpwDesignField::CoolingEnthalpyMeanCoincidentDryBulb0pt4)]);
+  setCoolingEnthalpy1(stringValues[index(EpwDesignField::CoolingEnthalpy1)]);
+  setCoolingEnthalpyMeanCoincidentDryBulb1(stringValues[index(EpwDesignField::CoolingEnthalpyMeanCoincidentDryBulb1)]);
+  setCoolingEnthalpy2(stringValues[index(EpwDesignField::CoolingEnthalpy2)]);
+  setCoolingEnthalpyMeanCoincidentDryBulb2(stringValues[index(EpwDesignField::CoolingEnthalpyMeanCoincidentDryBulb2)]);
+  if (m_ashraeHoFVersion <= ASHRAEHoFVintage::ASHRAE_2013) {
+    setCoolingHours8To4AndDryBulb12pt8To20pt6(stringValues[index(EpwDesignField::CoolingHours8To4AndDryBulb12pt8To20pt6)]);
+  } else {
+    setExtremeMaxWetBulb(stringValues[index(EpwDesignField::CoolingExtremeMaxWetBulb) - 1]);
+  }
```

Comment:
assignCoolingData: depending on vintage, the last field is CoolingHours8To4AndDryBulb12pt8To20pt6 or CoolingExtremeMaxWetBulb (notice the index -1 here)